android/cli/commands/_build.js

```diff
@@ -2796,6 +2796,38 @@ AndroidBuilder.prototype.copyResources = function copyResources(next) {
 				}
 			};
 		}), function () {
+			// jsanalyze will copy polyfils into buildBinAssetsResourcesDir and we need
+			// to unmark them here so they won't get removed on subsequent builds
+			if (this.transpile) {
+				/**
+				 * Recursively unmarks a module and it's dependencies from the last
+				 * build files list.
+				 *
+				 * @param {String} moduleId The module to remove from the list of files
+				 * @param {String} nodeModulesPath Path to the node_modules folder
+				 */
+				const unmarkPackageAndDependencies = (moduleId, nodeModulesPath) => {
+					let packageJsonPath;
+					if (require.resolve.paths) {
+						packageJsonPath = require.resolve(path.join(moduleId, 'package.json'), { paths: [ nodeModulesPath ] });
+					} else {
+						packageJsonPath = require.resolve(path.join(moduleId, 'package.json'));
+						packageJsonPath = path.join(nodeModulesPath, packageJsonPath.substring(packageJsonPath.indexOf('node_modules') + 12));
+					}
+					const modulePath = path.dirname(packageJsonPath);
+					Object.keys(this.lastBuildFiles).forEach(p => {
+						if (p.startsWith(modulePath)) {
+							delete this.lastBuildFiles[p];
+						}
+					});
+					const packageJson = fs.readJSONSync(packageJsonPath);
```

Comment:
Build is failing at this line. 
`fs` does not seem to have `readJSONSync`, `fs-extra` does.

```
2019-01-16T22:04:03.137Z | ERROR  | An uncaught exception was thrown!
fs.readJSONSync is not a function
2019-01-16T22:04:03.138Z | ERROR  | fs.readJSONSync is not a function
```